test/integration/verify/controls/azure_redis_caches.rb

```diff
@@ -0,0 +1,14 @@
+resource_group_name = attribute(:resource_group, default: nil)
+
+control 'azure redis caches test' do
+  describe azure_redis_caches(resource_group: resource_group_name) do
+    it { should exist }
+    its('sku_names') { should include 'Standard' }
+  end
+end
+
+control 'azure redis caches filters' do
+  describe azure_redis_caches(resource_group: resource_group_name).where(enable_non_ssl_port: true) do
```

Comment:
After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
it is not mandatory..